docs/source/framework/pytorch_integration/layers_database.rst

```diff
@@ -61,6 +61,16 @@ Simple Convolution
         O(n, m, h, w)  =  O(n, m, h, w) + B(m)
     }
 
+1D Convolution
+^^^^^^^^^^^^^^
+
+.. code::
+
+    def conv1d(float(N, C, L) I, float(M, C, KL) W1, float(M) B) -> (O) {
```

Comment:
`float(M, C, KL) W1` -> `float(F, C, KL) W1` ?

I seem to recall some soft ML conventions along the lines of using either the output/input channels  terminology or the filter/channel terminology.
If not, that works fine too, I just thought I'd ask